core/sql/exp/exp_datetime.cpp

```diff
@@ -91,6 +91,50 @@ copyDatetimeFields(rec_datetime_field startField,
                    char *dstData,
                    NABoolean *roundedDownFlag);
 
+//////////////////////////////////////////////
+// Defined in exp_datetime.h
+//
+//  struct DatetimeFormatInfo
+//  {
+//    Lng32 format;
```

Comment:
Could change the datatype here to an enum, with a slight improvement in type saftey